lib/Elastica/Client.php

```diff
@@ -62,19 +69,23 @@ class Client
      * @var LoggerInterface
      */
     protected $_logger = null;
+
     /**
-     * @var Connection\ConnectionPool
+     * @var BuilderInterface
      */
-    protected $_connectionPool = null;
```

Comment:
This breaks BC in case someone extended the Client. Not something that prevents us from moving forward, but worth mentioning in the CHANGELOG.md.